pkg/controllers/disruption/orchestration/metrics.go

```diff
@@ -43,11 +43,11 @@ var (
 			Buckets:   metrics.DurationBuckets(),
 		},
 	)
-	disruptionReplacementNodeClaimFailedCounter = prometheus.NewCounterVec(
+	disruptionQueueFailedCounter = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Namespace: metrics.Namespace,
 			Subsystem: disruptionSubsystem,
-			Name:      "replacement_nodeclaim_failures_total",
+			Name:      "queue_failures_total",
 			Help:      "The number of times that Karpenter failed to launch a replacement node for disruption. Labeled by disruption method.",
```

Comment:
Let's change the description too
```suggestion
			Help:      "The number of times that an enqueued disruption decision failed. Labeled by disruption method.",
```